File: src/Java/1. First Course/4. Third Hole/src/main/refactoringgolf/store/AccessoriesDiscount.java
package refactoringgolf.store;

public class AccessoriesDiscount implements CategoryDiscount {

	public float calculateDiscount(OrderItem orderItem) {
		float discount = 0;
		float unitPricePerQuantity = orderItem.unitPricePerQuantity();
		if (unitPricePerQuantity >= 100) {
			discount = unitPricePerQuantity * 10 / 100;
		}
		return discount;
	}

}

File: src/Net/1. First Course/5. Fourth Hole/RefactoringGolf.Store/AccessoriesDiscount.cs
namespace RefactoringGolf.Store
{
    public class AccessoriesDiscount : ICategoryDiscount
    {
        public decimal CalculateDiscount(OrderItem orderItem)
        {
            decimal discount = 0;
            if (orderItem.UnitPricePerQuantity() >= 100)
            {
                discount = orderItem.UnitPricePerQuantity() * 10 / 100;
            }
            return discount;
        }
    }
}